elasticsearch-singledocument/src/test/resources/log4j.xml

```diff
@@ -14,10 +14,6 @@
         </layout>
     </appender>
 
-    <logger name="org.vertexium.query.Query.QUERY" additivity="false">
```

Comment:
Instead of removing these lines should we comment them out with a comment stating something about to debug query you can turn this on or something. Or probably better yet put it in the ES readme. I always end up having to look up how to do this.